src/rsa.rs

```diff
@@ -89,6 +89,8 @@ struct OtherPrimeInfo<'a> {
     coefficient: DerEncBigUint<'a>,
 }
 
+// TODO Is this correct? According to the PKCS#11 specs, for wrapping we should use
+// BER-encoded according to PKCS #8’s PrivateKeyInfo ASN.1 type
```

Comment:
DER is a strict subset of BER encoding, so when we produce the PrivateKeyInfo we are BER compliant.
Unfortunately we may fail to parse a structure that actually uses BER encodings that are not conformant to the stricter DER, however I am not aware of any implementation that uses non-DER for encoding this structure. If that happens and we need to become compatible we'll have to figure out then what we want to do.

Reply:
the point here is the PrivateKeyInfo x RSAPrivateKey

Reply:
https://www.ietf.org/rfc/rfc5208.html#section-5
I see now, this is incomplete

Reply:
Tried to wrap the RSA with the PrivateKeyInfo structure, but it needs some more sharing between algorithms, which I did not do yet. And integration testing, which will be possible likely as soon as we will be able to use the module without the pReserved pointer. Resolving for now.